packages/ui-library/src/components/toggle-field/index.js

```diff
@@ -31,11 +31,12 @@ const ToggleField = forwardRef( ( {
 }, ref ) => (
 	<Switch.Group
 		as="div"
+
 		className={ classNames( "yst-toggle-field", disabled && "yst-toggle-field--disabled", className ) }
 	>
 		<div className="yst-toggle-field__header">
 			{ label && <div className="yst-toggle-field__label-wrapper">
-				<Label as={ Switch.Label } className="yst-toggle-field__label" label={ label } />
+				<Label as={ Switch.Label } className="yst-toggle-field__label" label={ label } aria-label={ props[ "aria-label" ] } />
```

Comment:
Let's pull the `aria-label` into the props instead of retrieving them from the props like this.
That provides some clarity for the interface.

Reply:
Done 👍